### PR TITLE
Brush up Kobold Breath damage automation

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: packs/feats/kobold-breath.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>You channel your draconic exemplar's power into a gout of energy that manifests as a @Template[type:line|distance:30] or a @Template[type:cone|distance:15], dealing 1d4 damage. Each creature in the area must attempt a basic saving throw against the higher of your class DC or spell DC. You can't use this ability again for [[/r 1d4 #rounds]]{1d4 rounds}.</p>\n<p>At 3rd level and every 2 levels thereafter, the damage increases by 1d4. The shape of the breath, the damage type, and the saving throw match those of your @UUID[Compendium.pf2e.ancestryfeatures.Item.Draconic Exemplar]. This ability has the trait associated with the type of damage it deals.</p>\n<hr />\n<p>@Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)|basic:true] @Check[type:reflex|dc:resolve(@actor.attributes.classOrSpellDC.value)|basic:true]</p>\n<p>@Damage[ceil(@actor.level/2)d4[@actor.flags.pf2e.draconicExemplar.damageType]]{Leveled damage}</p>"
```
Comment (TikaelSol) [inline packs/feats/kobold-breath.json:14]:
```suggestion
            "value": "<p>You channel your draconic exemplar's power into a gout of energy that manifests as a @Template[type:line|distance:30] or a @Template[type:cone|distance:15], dealing @Damage[ceil(@actor.level/2)d4[@actor.flags.pf2e.draconicExemplar.damageType]] damage. Each creature in the area must attempt a basic saving throw against the higher of your class DC or spell DC. You can't use this ability again for [[/r 1d4 #rounds]]{1d4 rounds}.</p>\n<p>At 3rd level and every 2 levels thereafter, the damage increases by 1d4. The shape of the breath, the damage type, and the saving throw match those of your @UUID[Compendium.pf2e.ancestryfeatures.Item.Draconic Exemplar]. This ability has the trait associated with the type of damage it deals.</p>\n<hr />\n<p>@Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)|basic:true] @Check[type:reflex|dc:resolve(@actor.attributes.classOrSpellDC.value)|basic:true]</p>"
```

File: packs/feats/dragonblood-paragon.json [hunk 2]
```diff
@@ -59,10 +59,22 @@
...
+                "key": "Note",
+                "predicate": [
+                    "dragonblood-paragon:kobold-breath"
+                ],
+                "selector": [
+                    "kobold-breath-inline-damage",
+                    "dragons-breath-inline-damage"
+                ],
+                "text": "Creatures that critically fail their save against your Kobold Breath take @Damage[3d4[@actor.flags.pf2e.draconicExemplar.damageType,persistent]]{3d4 persistent damage} of the type dealt by your draconic patron.",
+                "title": "Dragonblood Paragon"
```
[the remark this answers is not in the document]
Comment (pedrogrullada) [reply packs/feats/dragonblood-paragon.json]: First time messing with the localization files, not sure about standard practice here, but it's done

File: packs/feats/dragonblood-paragon.json [hunk 3]
```diff
@@ -59,10 +59,22 @@
...
+                "key": "Note",
+                "predicate": [
+                    "dragonblood-paragon:kobold-breath"
+                ],
+                "selector": [
+                    "kobold-breath-inline-damage",
+                    "dragons-breath-inline-damage"
+                ],
+                "text": "PF2E.SpecificRule.Kobold.DragonbloodParagon.KoboldBreath.Note",
+                "title": "Dragonblood Paragon"
```
Comment (TikaelSol) [inline packs/feats/dragonblood-paragon.json:77]:
```suggestion
                "title": "{item|name}"
```
Basically the idea is we never have displayed text like that in RE fields to make life easier for translation modules.

File: packs/ancestryfeatures/draconic-exemplar.json [hunk 4]
```diff
@@ -30,49 +30,85 @@
                 "choices": [
                     {
                         "label": "PF2E.SpecificRule.DragonDisciple.DragonChoice.Black",
-                        "value": "black"
+                        "value": {
+                            "damageType": "acid",
+                            "dragon": "black"
+                        }
```
Comment (TikaelSol) [inline packs/ancestryfeatures/draconic-exemplar.json:36]: Cue these all changing in 7 months :D

File: static/lang/re-en.json [hunk 5]
```diff
@@ -2409,7 +2409,10 @@
             "Kobold": {
                 "DragonbloodParagon": {
                     "Dracomancer": "Dracomancer",
-                    "KoboldBreath": "Kobold Breath",
+                    "KoboldBreath": {
+                        "Label": "Kobold Breath",
+                        "Note": "Creatures that critically fail their save against your Kobold Breath take @Damage[3d4[@actor.flags.pf2e.draconicExemplar.damageType,persistent]]{3d4 persistent damage} of the type dealt by your draconic patron."
```
Comment (TikaelSol) [inline static/lang/re-en.json:2414]:
```suggestion
                        "Note": "Creatures that critically fail their save against your Kobold Breath take @Damage[3d4[@actor.flags.pf2e.draconicExemplar.damageType,persistent]]."
```

File: packs/ancestryfeatures/draconic-exemplar.json (hunk 4 above)
Comment (pedrogrullada) [reply packs/ancestryfeatures/draconic-exemplar.json:36]: Didn't think about that... oh well.

**Pull request merged**